CacheWarmer/MetadataCacheWarmer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Bundle\MongoDBBundle\CacheWarmer;
+
+use Doctrine\Persistence\ObjectManager;
+use Symfony\Bundle\FrameworkBundle\CacheWarmer\AbstractPhpFileCacheWarmer;
+use Symfony\Component\Cache\Adapter\ArrayAdapter;
+
+use function is_file;
+
+/** @internal */
+final class MetadataCacheWarmer extends AbstractPhpFileCacheWarmer
```

Comment:
I'll propose adding this to the doctrine-bridge in Symfony so we can drop this class. In the interest of "getting on with it", I decided to first add it here since this will only be in 5.4.